app/storage/metadata_parser.py

```diff
@@ -47,22 +52,24 @@ def clean_leading_trailing_spaces(metadata):
     'boolean': boolean_parser,
     'string': string_parser,
     'optional_string': optional_string_parser,
+    'optional_uuid': optional_uuid_4_parser,
 }
 
-MANDATORY_METADATA = [
+METADATA_VALIDATION_RULES = [
```

Comment:
We should really rebase against Rich's changes as this is overlapping and has changed.

Reply:
Those changes aren't yet approved right?

Reply:
Nope, I meant more when it is in?.

Reply:
Sure, but I can't see how that's different to our usual workflow.

Reply:
I mean from the point that this card has overlaps with Rich's and maybe should have been on hold, as reviewing metadata parser is essentially redundant if/when Rich's goes in?.

Reply:
The card was originally on hold, and subsequently marked as unblocked - I consulted with Rich on whether he felt it ready to start. Yes they overlap, but then as I understand it, this card was originally marked the higher priority.